CONTRIBUTING

```diff
@@ -0,0 +1,9 @@
+How to contribute to MPICH
+
+1. Sign and subnit a contributors' agreement. http://www.mpich.org/documentation/contributor-docs/
+
+2. Format your patches according to the MPICH coding standards. https://wiki.mpich.org/mpich/index.php/Coding_Standards
```

Comment:
Those standards might not be up to date. I'd mention that we have a script to enforce these things.